[8.0][FIX] account_asset_management

rven opened the pull request:
When pressing the button "Change Duration" on a confirmed asset, the following error is shown:

  File "../account_asset_management/wizard/account_asset_change_duration.py", line 67, in fields_view_get 
    node_me = doc.xpath("//field[@name='method_end']")[0] 
IndexError: list index out of range

rven committed:
[FIX] account_asset_management: Error on change duration

  File "../account_asset_management/wizard/account_asset_change_duration.py", line 67, in fields_view_get 
    node_me = doc.xpath("//field[@name='method_end']")[0] 
IndexError: list index out of range

luc-demeyer commented:
@rven 
thanks raf.
I suggest to resubmit the PR (after signing CLA) but with less changes.
Just replace line 57 by 
        if active_model == 'account.asset.asset' and asset_id \
                and view_type == 'form':

rven committed:
[FIX] account_asset_management: Error on change duration

rven committed:
[8.0][FIX] account_asset: entries from assets

When an entry is generated from an asset, the number of this entry must be
the sequence number linked to the journal of the asset and the reference of
this entry is the name of the asset.

opw:639760

SHA 443605ce (open)
by Goffin Simon, 05/25/2015 06:35 PM
parent 5c690d37
